### ISSUE TITLE
latest support

### ISSUE THREAD

**Issue by meeftah** [Germanaz0/laravel-ckan-api#4]
in this version, is support the ckan latest api version and laravel 9?

**Comment by VTrace** [comment]
No, it doesn't. Hope there will be any update for latest laravel version.

**Comment by german-bortoli** [comment]
Thanks for reporting it,Going to work on a new upgrade soon On 23 Mar 2023, at 13:18, Valdie Setiawan ***@***.***> wrote:﻿
No, it doesn't. Hope there will be any update for latest laravel version.
